runtime/vm/OutOfLineINL_openj9_internal_foreign_abi_InternalUpcallHandler.cpp

```diff
@@ -119,15 +120,20 @@ OutOfLineINL_openj9_internal_foreign_abi_InternalUpcallHandler_allocateUpcallStu
 		goto freeAllMemoryThenExit;
 	}
 
-	/* The resolution is performed by MethodHandleResolver.upcallLinkCallerMethod()
+	/* The resolution is performed by MethodHandleResolver.ffiCallLinkCallerMethod()
 	 * to obtain a MemberName object plus appendix in a two element object array.
 	 */
 	VM_OutOfLineINL_Helpers::buildInternalNativeStackFrame(currentThread, method);
-	resolveUpcallInvokeHandle(currentThread, upcallMetaData);
+	invokeCache = resolveFfiCallInvokeHandle(currentThread,
+			J9VMOPENJ9INTERNALFOREIGNABIUPCALLMHMETADATA_CALLEEMH(currentThread,
+					J9_JNI_UNWRAP_REFERENCE(upcallMetaData->mhMetaData)));
```

Comment:
Each argument should begin on a new line; also lines 135-136.

Reply:
Adjusted everywhere.